trackintel/analysis/metrics.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+from tqdm import tqdm
+
+from trackintel.geogr import point_haversine_dist, check_gdf_planar
+
+
+def radius_gyration(gdf, method="count", print_progress=False):
+    """
+    Radius of gyration for individual users.
+
+    Parameters
+    ----------
+    gdf : Geodataframe
+        GeoDataFrame with column "user_id" and Point geometry.
```

Comment:
In trackintel, Rg only makes sense on the staypoint level - can we specify it here?

Reply:
I changed it to `Staypoints` everywhere. 
But isn't this metric applicable to Positionfixes and Locations as well?

Reply:
It applies to any traces, but the positions shall be with some meanings - I would therefore argue it does not apply for pfs. Locs do not contain frequency information - two visits to the same loc will only be saved once in the locs table - which is also not rg is measuring.

Reply:
okay, thank you for answering :)